### PR TITLE
Update VS 2022 container image blueprints to .NET 6

### PR DESCRIPTION
**Pull request by normj**
*Description of changes:*
Update all of the VS 2022 container image blueprints to .NET 6 in anticipation of the base image being released.

Changes:
* Update all template.json files to call out .NET 6. This file is used as the description in the .NET CLI `new` command
* Update all blueprint-manifest.json files to call out .NET 6. This file is used as the description in VS project wizard
* Update references of public.ecr.aws/lambda/dotnet:5.0 to public.ecr.aws/lambda/dotnet:6.0
* Update references of mcr.microsoft.com/dotnet/sdk:5.0-buster-slim to mcr.microsoft.com/dotnet/sdk:6.0-bullseye-slim
* Update all `TargetFramework` elements to `net6.0`
* Remove ImageConfig.EntryPoint element in serverless.template. This was added for the initial release of image support and CloudFormation had an issue when only Command was set. This work around is no longer needed.
* In test projects update to latest test packages
* In F# test projects remove unnecessary `main` method since the test projects are class library. .NET 6 makes this a compile error to leave it in
* Make build script more resilient by force uninstalling the templates before installing and executing them.

The changes were tested by manually and running the build target `test-blueprints-dotnew` which creates each templates and make sure they compile.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Blueprints/BlueprintDefinitions/vs2022/AspNetCoreWebAPI-Image-FSharp/template/src/BlueprintBaseName.1/Readme.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ Alternatively the Docker file could be written to use [multi-stage](https://docs
 have the .NET project built inside the container. Below is an example of building .NET 5 project inside the image.
```
Comment (boblodgett) [inline Blueprints/BlueprintDefinitions/vs2022/AspNetCoreWebAPI-Image-FSharp/template/src/BlueprintBaseName.1/Readme.md:44]: .NET 5 => .NET 6

File: Blueprints/BlueprintDefinitions/vs2022/AspNetCoreWebAPI-Image/template/src/BlueprintBaseName.1/Readme.md [hunk 2]
```diff
@@ -51,9 +51,9 @@ Alternatively the Docker file could be written to use [multi-stage](https://docs
 have the .NET project built inside the container. Below is an example of building .NET 5 project inside the image.
```
Comment (boblodgett) [inline Blueprints/BlueprintDefinitions/vs2022/AspNetCoreWebAPI-Image/template/src/BlueprintBaseName.1/Readme.md:51]: .NET 5 => .NET 6

File: Blueprints/BlueprintDefinitions/vs2022/EmptyFunction-Image-FSharp/template/src/BlueprintBaseName.1/Readme.md [hunk 3]
```diff
@@ -22,9 +22,9 @@ Alternatively the Docker file could be written to use [multi-stage](https://docs
 have the .NET project built inside the container. Below is an example of building .NET 5 project inside the image.
```
Comment (boblodgett) [inline Blueprints/BlueprintDefinitions/vs2022/EmptyFunction-Image-FSharp/template/src/BlueprintBaseName.1/Readme.md:22]: .NET 5 => .NET 6
or .NET alone may be better.

File: Blueprints/BlueprintDefinitions/vs2022/EmptyFunction-Image/template/src/BlueprintBaseName.1/Readme.md [hunk 4]
```diff
@@ -22,9 +22,9 @@ Alternatively the Docker file could be written to use [multi-stage](https://docs
 have the .NET project built inside the container. Below is an example of building .NET 5 project inside the image.
```
Comment (boblodgett) [inline Blueprints/BlueprintDefinitions/vs2022/EmptyFunction-Image/template/src/BlueprintBaseName.1/Readme.md:22]: .NET 5 => .NET 6

File: Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image-FSharp/template/src/BlueprintBaseName.1/Readme.md [hunk 5]
```diff
@@ -24,9 +24,9 @@ Alternatively the Docker file could be written to use [multi-stage](https://docs
 have the .NET project built inside the container. Below is an example of building .NET 5 project inside the image.
```
Comment (boblodgett) [inline Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image-FSharp/template/src/BlueprintBaseName.1/Readme.md:24]: .NET 5 => .NET 6

File: Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image/template/src/BlueprintBaseName.1/Readme.md [hunk 6]
```diff
@@ -24,9 +24,9 @@ Alternatively the Docker file could be written to use [multi-stage](https://docs
 have the .NET project built inside the container. Below is an example of building .NET 5 project inside the image.
```
Comment (boblodgett) [inline Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image/template/src/BlueprintBaseName.1/Readme.md:24]: .NET 5 => .NET 6

File: buildtools/build.proj [hunk 7]
```diff
@@ -99,6 +100,7 @@
         <MSBuild Projects="$(MSBuildProjectFile)" Targets="run-blueprint-dotnetnew" Properties="TemplateName=serverless.StepFunctionsHelloWorld;ProjectName=StepFunctionsHelloWorldC;Lang=C#;Version=vs2022"/>
         <MSBuild Projects="$(MSBuildProjectFile)" Targets="run-blueprint-dotnetnew" Properties="TemplateName=serverless.StepFunctionsHelloWorld;ProjectName=StepFunctionsHelloWorldF;Lang=F#;Version=vs2022"/>				
 		
+		<Exec Command="dotnet new -u Blueprints\BlueprintDefinitions\vs2019" WorkingDirectory="..\" IgnoreExitCode="true" />
```
Comment (boblodgett) [inline buildtools/build.proj:103]: nit: indentation problems in this file. Do not need to fix.

**Review by boblodgett — APPROVED** [review]

File: Blueprints/BlueprintDefinitions/vs2022/AspNetCoreWebAPI-Image-FSharp/template/src/BlueprintBaseName.1/Readme.md (hunk 1 above)
Comment (normj) [reply Blueprints/BlueprintDefinitions/vs2022/AspNetCoreWebAPI-Image-FSharp/template/src/BlueprintBaseName.1/Readme.md:44]: Fixed

File: Blueprints/BlueprintDefinitions/vs2022/AspNetCoreWebAPI-Image/template/src/BlueprintBaseName.1/Readme.md (hunk 2 above)
Comment (normj) [reply Blueprints/BlueprintDefinitions/vs2022/AspNetCoreWebAPI-Image/template/src/BlueprintBaseName.1/Readme.md:51]: Fixed

File: Blueprints/BlueprintDefinitions/vs2022/EmptyFunction-Image-FSharp/template/src/BlueprintBaseName.1/Readme.md (hunk 3 above)
Comment (normj) [reply Blueprints/BlueprintDefinitions/vs2022/EmptyFunction-Image-FSharp/template/src/BlueprintBaseName.1/Readme.md:22]: Fixed

File: Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image-FSharp/template/src/BlueprintBaseName.1/Readme.md (hunk 5 above)
Comment (normj) [reply Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image-FSharp/template/src/BlueprintBaseName.1/Readme.md:24]: Fixed

File: Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image/template/src/BlueprintBaseName.1/Readme.md (hunk 6 above)
Comment (normj) [reply Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image/template/src/BlueprintBaseName.1/Readme.md:24]: Fixed

File: buildtools/build.proj (hunk 7 above)
Comment (normj) [reply buildtools/build.proj:103]: Fixed

**Review by boblodgett — APPROVED** [review]

**Pull request merged**